File: Level-0/76.java
class Solution {
    public String solution(String cipher, int code) {
        String answer = "";
        for(int i = 1; i <= cipher.length() / code; i++){
            char c = cipher.charAt(i * code - 1);
            answer += String.valueOf(c);
        }
        return answer;
    }
}

File: Level-0/76.js
function solution(cipher, code) {
    const arr = cipher.split("");
    let answer = "";
    for(let i = 0; i < arr.length; i++) {
        if((i + 1) % code === 0) {
            answer += arr[i];
        }
    }
    return answer;
}
